build.gradle

```diff
@@ -241,7 +241,7 @@ dependencies {
 
   compile group: 'uk.gov.hmcts.reform', name: 'core-case-data-store-client', version: '4.7.6'
   compile group: 'com.github.hmcts', name: 'doc-assembly-client', version: '1.2.2'
-  compile group: 'com.github.hmcts', name: 'fees-java-client', version: '0.0.6'
+  compile group: 'uk.gov.hmcts.reform', name: 'doc-assembly-client', version: '1.07'
```

Comment:
Please use this: compile group: 'com.github.hmcts', name: 'doc-assembly-client', version: '1.2.2' and above, Maven has outdated one..

Reply:
Otherwise LGTM